documentation/book/proc-configuring-kafka-mirror-maker-authentication-scram-sha-512.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='proc-configuring-kafka-mirror-maker-authentication-scram-sha-512-{context}']
+= Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator with a `listener` configured for SCRAM-SHA-512 authentication
+* Username to be used for authentication
+
+.Procedure
+
+Because Kafka Mirror Maker connects to two Kafka clusters (source and target), it is possible to configure SCRAM-SHA-512 authentication for one or both of them.
```

Comment:
Rewrite to:

As the Kafka Mirror Maker connects to two Kafka clusters (source and target), you can choose to configure SCRAM-SHA-512 authentication for one or both the clusters. The following steps describe how to configure SCRAM-SHA-512 authentication on the consumer side for connecting to the source Kafka cluster: